install/container/ubuntu.sh

```diff
@@ -45,6 +43,12 @@ if [ ${DISTRIB_RELEASE_MAJOR} -le 20 ]; then
   locale-gen "en_US.UTF-8"
 fi
 
+# INCLUDE+ common/Dockerfile.apt-install-required-before-2204
+if [ ${DISTRIB_RELEASE_MAJOR} -le 20 ]; then
+  apt-get -y update && apt-get install -y \
+    python 
+fi
```

Comment:
Follow-up lets put all python related installation in the same place with an else clause (python3 work on -ge 20 https://github.com/tipi-build/cli/pull/108/files#diff-6920cfd1fbd498a855ffa6f78893bb11c61882fdff3af45902afc3845879fbe6L73-R81)